release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -131,6 +131,46 @@ For more information, see xref:../networking/openshift_network_security/configur
 [id="ocp-4-16-storage"]
 === Storage
 
+[id="ocp-4-16-storage-clone-azure-file"]
+==== Volume cloning supported for Azure File (Technology Preview)
+{product-title} 4.16 introduces volume cloning for the Microsoft Azure File Container Storage Interface (CSI) Driver Operator as a Technology Preview feature. Volume cloning duplicates an existing persistent volume (PV) to help protect against data loss in {product-title}.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-azure-file.adoc#persistent-storage-csi-azure-file[Azure File CSI Driver Operator] and xref:..//storage/container_storage_interface/persistent-storage-csi-cloning.adoc#persistent-storage-csi-cloning[CSI volume cloning].
+
+[id="ocp-4-16-storage-node-expansion-secret"]
+==== Node Expansion Secret is generally available
+The Node Expansion Secret feature allows your cluster to expand storage volumes, even when access to those volumes requires a secret (for example, a credential for accessing a Storage Area Network (SAN) fabric) to perform the node expand operation. {product-title} 4.16 supports this feature as generally available.
```

Comment:
I think it's for "mounted" volumes.
Suggest: `The Node Expansion Secret feature allows your cluster to expand storage mounted volumes`
cc @jsafrane

Reply:
Fixed.